### ISSUE TITLE
Sommelier not working on arm device

### ISSUE THREAD

**Issue by justinkkwan** [chromebrew/chromebrew#7224]
## Description

After installing Sommelier through Chromebrew on my Lenovo 10e, Sommelier can not start.

If you executed a specific command:
```shell
crew install sommelier
source ~/.bashrc
```

Output:
```
some sommelier processes failed to start
sommelier processess running:

```

## Versions and system information

***
fatal: ambiguous argument 'HEAD^..HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
- Architecture: `aarch64` (`aarch64`)
- Kernel version: `5.10.114-16025-ge75506b9d98e`

- Chromebrew version: `1.24.0`
- Chromebrew prefix: `/usr/local`
- Chromebrew libdir: `/usr/local/lib`

- Last update in local repository: ``

- OS variant: `Chrome OS`
- OS version: `kukui-release/R103-14816.131.0`
- OS channel: `stable-channel`
***

## Additional information

After this failed, I tried `crew reinstall sommelier` but that didn't help.
Then I did powerwashed (twice, actually) and did nothing other than the installation steps from the readme and installing sommelier.
When it still failed, I tried another `crew reinstall sommelier` which did not help.
In every case, the same thing I pasted above was printed to the console.

**Comment by uberhacker** [comment]
@justinkkwan: Can you please provide the output from `/usr/local/var/log/sommelier.log`?

**Comment by justinkkwan** [comment]
Contents of `/usr/local/var/log/sommelier.log`:
```
MESA-LOADER: failed to open vgem: /usr/local/lib/dri/vgem_dri.so: cannot open shared object file: No such file or directory (search paths /usr/local/lib/dri, suffix _dri)
failed to load driver: vgem
MESA-LOADER: failed to open kms_swrast: /usr/local/lib/libLLVM-14.so: cannot open shared object file: No such file or directory (search paths /usr/local/lib/dri, suffix _dri)
failed to load driver: kms_swrast
MESA-LOADER: failed to open swrast: /usr/local/lib/libLLVM-14.so: cannot open shared object file: No such file or directory (search paths /usr/local/lib/dri, suffix _dri)
failed to load swrast driver
error: couldn't get display device
MESA-LOADER: failed to open vgem: /usr/local/lib/dri/vgem_dri.so: cannot open shared object file: No such file or directory (search paths /usr/local/lib/dri, suffix _dri)
failed to load driver: vgem
MESA-LOADER: failed to open kms_swrast: /usr/local/lib/libLLVM-14.so: cannot open shared object file: No such file or directory (search paths /usr/local/lib/dri, suffix _dri)
failed to load driver: kms_swrast
MESA-LOADER: failed to open swrast: /usr/local/lib/libLLVM-14.so: cannot open shared object file: No such file or directory (search paths /usr/local/lib/dri, suffix _dri)
failed to load swrast driver
error: couldn't get display device
MESA-LOADER: failed to open vgem: /usr/local/lib/dri/vgem_dri.so: cannot open shared object file: No such file or directory (search paths /usr/local/lib/dri, suffix _dri)
failed to load driver: vgem
MESA-LOADER: failed to open kms_swrast: /usr/local/lib/libLLVM-14.so: cannot open shared object file: No such file or directory (search paths /usr/local/lib/dri, suffix _dri)
failed to load driver: kms_swrast
MESA-LOADER: failed to open swrast: /usr/local/lib/libLLVM-14.so: cannot open shared object file: No such file or directory (search paths /usr/local/lib/dri, suffix _dri)
failed to load swrast driver
error: couldn't get display device
```

**Comment by uberhacker** [comment]
Maybe try:
```
crew reinstall llvm mesa
crew reinstall sommelier -s
```

**Comment by justinkkwan** [comment]
During the `crew reinstall llvm mesa`, I got:
```
Removing since upgrade or reinstall...
Package llvm isn't installed.
```

But alas, after `crew reinstall sommelier -s` and `source ./bashrc` I got the same:
```
some sommelier processes failed to start
sommelier processes running:

```

Additionally, these three lines were added to the end of `/usr/local/var/log/sommelier.log`:

```
MESA-LOADER: failed to open vgem: /usr/local/lib/dri/vgem_dri.so: cannot open shared object file: No such file or directory (search paths /usr/local/lib/dri, suffix _dri)
failed to load driver: vgem
/usr/local/bin/Xwayland: error while loading shared libraries: /usr/local/lib/libxcvt.so.0: cannot open shared object file: No such file or directory
```

**Comment by satmandu** [comment]
Does `crew reinstall libxcvt` help?

**Comment by justinkkwan** [comment]
Yep, that seemed to do the trick. And just like with llvm mesa, reinstalling said it was never installed in the first place. Perhaps these 2 packages need to be added as dependencies for arm?

**Comment by satmandu** [comment]
@uberhacker did those deps get lost on that sommelier reversion a couple of weeks ago?

**Comment by uberhacker** [comment]
No, llvm should be installed via core and mesa is already a dependency so not sure why those were missing.  I did add the missing libxcvt dependency in #7225.

**Comment by uberhacker** [comment]
Fixed via #7225.
